config/core/test-configs.yaml

```diff
@@ -2388,6 +2394,11 @@ test_configs:
       - baseline
       - baseline-nfs
 
+  - device_type: kontron-kswitch-d10-mmt-8g
+    test_plans:
+      - baseline
+      - baseline-nfs
```

Comment:
This is failing with the defconfig + debug fragment:
https://staging.kernelci.org/test/plan/id/6314379cb17a43d91c98563b/

but passing with SMP=n:
https://staging.kernelci.org/test/plan/id/6314405fac38fe2c95985656/

I think it may just be a glitch in the lab, still it might be worth resubmitting the job with +debug fragment to see if it's always failing to boot or if it's intermittent and not related to the kernel config.

Reply:
@mwalle Could you please take a look?  It would be good to understand the reason for the NFS failure.  I don't think this will be blocking the PR, maybe we could add a filter to not run the +debug builds on this platform if that's the problem.

Reply:
Yes, I'll take a look. FWIW also the other kswitch has that problem:
https://lavalab.kontron.com/scheduler/job/163750

I guess it's related to the loadaddresses in u-boot and the actual kernel sizes.

Reply:
OK let us know if you make a fix for that.

Reply:
Hi,

sooo I'm not sure what's going on with these debug builds, but they are super slow to boot. As far as I see, none of the actual hardware boards are able to boot it. Only the QEMU ones.

In this particular case, the hardware watchdog will bite. I'm not sure if I can disable it in the bootloader. Is there a configuarion in lava to run extra commands in the bootloader before booting?

But even if I disable the watchdog, it takes about 230s to reach the prompt which is very near to the overall job timeout.

-michael